### DISCUSSION TITLE
Student Developer pack: not near campus location

### DISCUSSION THREAD

**Discussion by Shrey-12** [community/community#40689] [GitHub Education]
Despite entering all details and uploading a valid id card proof and being present in my college's campus, it keeps showing "not near campus location", How do I fix this?

**Comment by samus-aran** [comment]
Hi @Shrey-12 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.
